### ISSUE TITLE
Missing lyrics for OK computer remaster.

### ISSUE THREAD

**Issue by DawarTheDastardly** [surfbryce/beautiful-lyrics#500]
Songs from the remastered version of OK computer show up as not having lyrics, while the ones from the original album show lyrics as normal.

- **Original:**
![image](https://github.com/surfbryce/beautiful-lyrics/assets/89464296/185e3fc9-dc26-47ac-a378-bf3cf141ce70)

- **Remaster:**
![image](https://github.com/surfbryce/beautiful-lyrics/assets/89464296/21415a66-7a6a-45bb-9f49-9e758ea810db)

It does this for every song from the album except for paranoid android, which shows up as not having lyrics on both the original and the remaster.

**Comment by surfbryce** [comment]
@DawarTheDastardly Karma Police remastered should be working now, this was because I checked the dates when the song was released compared to what Spotify had. For some reason Spotify says 2017 and my service says 1997; so I've disabled that check for now and we'll see how that affects lyrics for other songs. But it seems ok and it got the lyrics for the remastered version.

Currently working on Paranoid Android, this is an issue with how I get the name for the song, should be a simple fix, you'll see all of this fixed in the next patch
